secboot/secboot_sb_test.go

```diff
@@ -1079,13 +1099,13 @@ func createMockSnapFile(snapDir, snapPath, snapType string) (snap.Container, err
 	return snapfile.Open(snapPath)
 }
 
-func mockSbTPMConnection(c *C, tpmErr error) (*sb.TPMConnection, func()) {
-	tcti, err := tpm2.OpenTPMDevice("/dev/null")
+func mockSbTPMConnection(c *C, tpmErr error) (*sb_tpm2.Connection, func()) {
+	tcti, err := linux.OpenDevice("/dev/null")
 	c.Assert(err, IsNil)
-	tpmctx, err := tpm2.NewTPMContext(tcti)
+	tpmctx := tpm2.NewTPMContext(tcti)
 	c.Assert(err, IsNil)
-	tpm := &sb.TPMConnection{TPMContext: tpmctx}
-	restore := secboot.MockSbConnectToDefaultTPM(func() (*sb.TPMConnection, error) {
+	tpm := &sb_tpm2.Connection{TPMContext: tpmctx}
+	restore := secboot.MockSbConnectToDefaultTPM(func() (*sb_tpm2.Connection, error) {
 		if tpmErr != nil {
 			return nil, tpmErr
 		}
```

Comment:
@chrisccoulson it's used here, inside a mock.

Reply:
Ah, I didn't see that - thanks!